crypto/evp/evp_extra_test.cc

```diff
@@ -521,6 +629,8 @@ TEST(EVPExtraTest, d2i_PrivateKey) {
                               sizeof(kExampleDSAKeyDER)));
   EXPECT_TRUE(ParsePrivateKey(EVP_PKEY_RSA, kExampleRSAKeyPKCS8,
                               sizeof(kExampleRSAKeyPKCS8)));
+  EXPECT_TRUE(ParsePrivateKey(EVP_PKEY_RSA_PSS, kExampleRSAPSSKeyPKCS8,
```

Comment:
What about a public key? We also implement the first part of `rsa_pss_pub_decode`.

Reply:
This has been covered by `X509Test.TestRsaSsaPss` in x509_test.cc. rsa_pss_pub_decode is to decode bytes to `SubjectPublicKeyInfo`.